src/java.base/share/classes/java/time/format/DateTimeFormatterBuilder.java

```diff
@@ -1148,14 +1148,14 @@ public DateTimeFormatterBuilder appendZoneId() {
      * During parsing, the text must match a known zone or offset.
      * There are two types of zone ID, offset-based, such as '+01:30' and
      * region-based, such as 'Europe/London'. These are parsed differently.
-     * If the parse starts with '+', '-', 'UT', 'UTC' or 'GMT', then the parser
-     * expects an offset-based zone and will not match region-based zones.
-     * The offset ID, such as '+02:30', may be at the start of the parse,
-     * or prefixed by  'UT', 'UTC' or 'GMT'. The offset ID parsing is
-     * equivalent to using {@link #appendOffset(String, String)} using the
-     * arguments 'HH:MM:ss' and the no offset string '0'.
-     * If the parse starts with 'UT', 'UTC' or 'GMT', and the parser cannot
-     * match a following offset ID, then {@link ZoneOffset#UTC} is selected.
+     * If the parse starts with '+' or '-', then the parser expects an
+     * offset-based zone and will not match region-based zones. The offset
+     * ID parsing is equivalent to using {@link #appendOffset(String, String)}
+     * using the arguments 'HH:MM:ss' and the no offset string '0'.
```

Comment:
The change to drop UT, UTC, GMT, from offset parsing, looks to be correct.
However, it does accept parsing of the offset formatted strings in contradiction to the 2nd paragraph above.
``` 
     * This appends an instruction to format/parse the zone ID to the builder
     * only if it is a region-based ID.
```
This pre-existing text is also isn't clear in that the formatter is always appended to the builder.
The accepting only of a region-based ID occurs when the DateTimeFormatter.parse(...) is parsing.
Perhaps a separate clarification may be useful

Reply:
Thanks, Roger. Updated the description per your suggestion.